libraries/AP_Scripting/examples/mission-load.lua

```diff
@@ -1,6 +1,8 @@
 -- Example of loading a mission from the SD card using Scripting
 -- Would be trivial to select a mission based on scripting params or RC switch
 -- luacheck: only 0
+--In case of testing it with SITL on linux - copy this "mission-load.lua" script to "ardupilot/ArduCopter/scripts" directory
```

Comment:
I dont have any issue adding this comment in the lua script once its been corrected, but its not necessary since we have an entire wiki page for running scripting in SITL in the wiki

Reply:
It is not about where the lua script should be. It is mentioned in the wiki page. But where the mission1.txt file should be is not clear from wiki. If anyone is using SITL or using MAVFTP to transfer file to sdcard... the directory where to copy this waypoint file(mission1.txt) needs to be mentioned. It took me hours to figure it out. On drone with Orange cube where you have no direct access to sdcard, I couldn't easily find where to copy this mission file using MAVFTP.

Reply:
then the text should say something like "put the mission1.txt file in the same directory as the script". Being absolute with the directory name makes this unnecessarily complicated

Reply:
Place "mission1.txt" in the directory one level above the "scripts" directory in case of SITL. In case of flight controller, it should be put at level as APM.
- APM
  - scripts
- mission1.txt

Reply:
I do no think this is correct..the lua io.open call expects the file (unless pathname is specified) to be from the same directory the script is called

Reply:
I tried and tested it and found the working solution that is mentioned in my previous message.It loads mission.txt file successfully from the location that I mentioned.